manifest.json

```diff
@@ -7,15 +7,17 @@
       "headers": [
         "x",
         "entry"
-      ]
+      ],
+      "format": "hostname"
```

Comment:
`domain` seems to better match the MDM schema (`.domain.domain`, `.domain.root_domain`), but nbd / defer to you and @rw-access

Reply:
oh, maybe we want to reserve `domain` for when it's an actually `Domain` struct? in which case, ya'll are already a step ahead

Reply:
Yeah I find `hostname` confusing too. It is part of the jsonschema specs, though. I don't know if different formats would break anything in our OpenAPI schema so for now, we're just going to stick with those. It's not enforced anywhere AFAICT, so I think we're free to change our minds in the future 
.https://json-schema.org/understanding-json-schema/reference/string.html#built-in-formats